HTTPShortcuts/app/src/main/kotlin/ch/rmy/android/http_shortcuts/activities/ExecuteActivity.kt

```diff
@@ -16,44 +19,18 @@ import ch.rmy.android.http_shortcuts.data.models.Base
 import ch.rmy.android.http_shortcuts.data.models.ResponseHandling
 import ch.rmy.android.http_shortcuts.data.models.Shortcut
 import ch.rmy.android.http_shortcuts.dialogs.DialogBuilder
-import ch.rmy.android.http_shortcuts.exceptions.CanceledByUserException
-import ch.rmy.android.http_shortcuts.exceptions.InvalidUrlException
-import ch.rmy.android.http_shortcuts.exceptions.ResumeLaterException
-import ch.rmy.android.http_shortcuts.exceptions.UnsupportedFeatureException
-import ch.rmy.android.http_shortcuts.exceptions.UserException
-import ch.rmy.android.http_shortcuts.extensions.cancel
-import ch.rmy.android.http_shortcuts.extensions.detachFromRealm
-import ch.rmy.android.http_shortcuts.extensions.finishWithoutAnimation
-import ch.rmy.android.http_shortcuts.extensions.logException
-import ch.rmy.android.http_shortcuts.extensions.mapFor
-import ch.rmy.android.http_shortcuts.extensions.mapIf
-import ch.rmy.android.http_shortcuts.extensions.showToast
-import ch.rmy.android.http_shortcuts.extensions.startActivity
-import ch.rmy.android.http_shortcuts.extensions.takeUnlessEmpty
-import ch.rmy.android.http_shortcuts.extensions.truncate
-import ch.rmy.android.http_shortcuts.extensions.type
-import ch.rmy.android.http_shortcuts.http.CookieManager
-import ch.rmy.android.http_shortcuts.http.ErrorResponse
-import ch.rmy.android.http_shortcuts.http.ExecutionScheduler
-import ch.rmy.android.http_shortcuts.http.FileUploadManager
-import ch.rmy.android.http_shortcuts.http.HttpRequester
-import ch.rmy.android.http_shortcuts.http.ShortcutResponse
+import ch.rmy.android.http_shortcuts.exceptions.*
+import ch.rmy.android.http_shortcuts.extensions.*
+import ch.rmy.android.http_shortcuts.http.*
```

Comment:
Please leave the imports as they are. I prefer to have them explicit, as it makes it easier to find out where a specific class or function comes from.